packages/react-dom/src/server/ReactDOMLegacyServerImpl.js

```diff
@@ -76,7 +76,7 @@ function renderToStringImpl(
   // That way we write only client-rendered boundaries from the start.
   abort(request, abortReason);
   startFlowing(request, destination);
-  if (didFatal) {
+  if (didFatal && fatalError !== abortReason) {
```

Comment:
Since we now fatal if we abort after suspending, I need to ignore this case to throw the error below. Although that error message doesn't make any sense so we should probably do something else there.

Reply:
Why does this fatal not make sense? Isn’t it just any error thrown in the shell?

Reply:
It’s just that 1) this is a string, not an Error 2) we have a different error message below intended for this case.